site/content/en/docs/Guides/Client SDKs/_index.md

```diff
@@ -264,6 +264,8 @@ to the capacity of that Counter or List.
 All functions will return an error if the specified `key` is not predefined in the 
 [`GameServer.Spec.Counters`][gameserverspec] resource configuration.
 
+**Note:** For Counters, the default settings for both capacity and count are preset to 1000. It is recommended to avoid configuring the capacity or count to max(int64), as doing so could cause problems with [JSON Patch operations](https://github.com/googleforgames/agones/issues/3636).
```

Comment:
I've updated the yaml and .md files to show the default setting of 1000 for counters. Can you please check if these updates are ok or if this `Note` section in the `Agones Game Server Client SDKs` documentation covers everything needed?

Reply:
Only the capacity defaults to 1000, the count defaults to 0.